fs2loader/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.fs2loader/Fs2Loader.scala

```diff
@@ -93,7 +93,7 @@ object Fs2Loader {
         goodSink(queue)
       )
       sinkTypes = dequeueTypes(queue)
-      _ <- sinkBadGood.concurrently(sinkTypes).compile.drain
+      _ <- sinkBadGood.compile.drain *> sinkTypes.compile.drain
```

Comment:
Just in case - `sinkBadGood` is a long running process. As we all `sinkTypes`. When you call `drain` you're making caller to wait until a process finished. In this case `sinkTypes` should never get executed